### ISSUE TITLE
Add bun as a package manager in the installation steps

### ISSUE THREAD

**Issue by ulises-jeremias** [Create-Node-App/create-node-app#25]
### Describe the feature

We want to add bun as a possible package manager

### Use Case

-

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

latest

### Environment details (OS name and version, etc.)

any

**Comment by VoidVampire** [comment]
Hi, I want to ask if only helpers.ts and installer.ts are the files necessary to implement this?

**Comment by ulises-jeremias** [comment]
> Hi, I want to ask if only helpers.ts and installer.ts are the files necessary to implement this?

Yes! And also this one! https://github.com/Create-Node-App/create-node-app/blob/0599b049098660320827f94d033ece71255ba873/packages/create-awesome-node-app/src/options.ts#L13

**Comment by ulises-jeremias** [comment]
@VoidVampire let me Know if you want me to assign this issue to you!

**Comment by VoidVampire** [comment]
Sorry to inform but I tried implementing it and failed to do so, if I ever get it to work I will again. 😓

**Comment by ulises-jeremias** [comment]
@VoidVampire we can pair on this if you want! would love to have you contributing on this 😊 feel free to send me your discord user or anything and I'll reach out to pair with you 😊 

I can speak english or spanish 👌🏻

**Comment by VoidVampire** [comment]
@ulises-jeremias Alright, I will contact you on mail.

**Comment by ulises-jeremias** [comment]
> @ulises-jeremias Alright, I will contact you on mail.

Sounds great! My email is ulisescf.24@gmail.com
